apps/member/src/pages/SupportListPage/components/SupportAnswerInput.tsx

```diff
@@ -0,0 +1,137 @@
+import { useCallback } from 'react';
+
+import { Button } from '@clab-platforms/design-system';
+import { cn } from '@clab-platforms/utils';
+
+import Textarea from '@components/common/Textarea/Textarea';
+
+import { useToast } from '@hooks/common/useToast';
+import {
+  useAnswerModifyMutation,
+  useAnswerWriteMutation,
+} from '@hooks/queries';
+
+import { SupportAnswerItem } from '@type/support';
+
+interface SupportAnswerInputProps {
+  onCancel?: () => void;
+  className?: string;
+  setAnswer: (answer: SupportAnswerItem) => void;
+  answer: SupportAnswerItem;
+  isAnswered?: boolean;
+}
+
+const SupportAnswerInput = ({
+  className,
+  onCancel,
+  setAnswer,
+  answer,
+  isAnswered = false,
+}: SupportAnswerInputProps) => {
+  const { addToast } = useToast();
+  const { content, id } = answer;
+
+  const { answerWriteMutate, isPending: isWritePending } =
+    useAnswerWriteMutation();
+  const { answerModifyMutate, isPending: isModifyPending } =
+    useAnswerModifyMutation();
```

Comment:
요거 어차피 가져와서 다시 `isWritePending`, `isModifyPending` 으로 네이밍 해 주어야 한다면, `mutate` 처럼 hook 반환 시에 네이밍된 상태로 반환해 주면 어떨까요? 
```suggestion
  const { answerWriteMutate, isWritePending } = useAnswerWriteMutation();
  const { answerModifyMutate, isModifyPending } = useAnswerModifyMutation();
```
그렇게 되면 이렇게 사용하면 될 것 같아요!